src/components/Router/index.js

```diff
@@ -209,25 +212,54 @@ class CrossDeviceMobileRouter extends Component {
     this.sendMessage('client success', { captures })
   }
 
+  isFallbackDisabledAndShouldUseCamera = (step) => {
```

Comment:
It would make the function name a bit verbose, but IMO would be clearer to devs to specify what fallback we're referring to here.